client/Dockerfile

```diff
@@ -46,8 +46,8 @@ ENV TIMEOUT=60
 ENV PHP_EXT_DIR=/usr/local/lib/php/extensions/no-debug-non-zts-20200930/
 
 # Install core PHP extensions
-RUN apk add --no-cache openssl nginx nginx-mod-http-headers-more su-exec postgresql-client libzip-dev unzip php8-pecl-igbinary php8-pecl-redis php8-pecl-imagick
-RUN docker-php-ext-install pcntl zip opcache
+RUN apk add --no-cache openssl nginx nginx-mod-http-headers-more su-exec postgresql-client libzip-dev unzip php8-pecl-igbinary php8-pecl-redis php8-pecl-imagick gmp gmp-dev
+RUN docker-php-ext-install pcntl zip opcache gmp
```

Comment:
This was required to ensure generating the public keys from JWKs happens quickly.